src/app/landing-page/landing-section/landing-section.component.scss

```diff
@@ -0,0 +1,86 @@
+.landing-section {
+  height: 90vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-image: url("/assets/landing-page-background.png");
+  background-repeat: no-repeat;
+  background-position-x: right;
+  background-size: cover;
+}
+
+.landing-section-text {
+  width: 70vw;
+  height: min-content;
+  display: grid;
+  grid-template-areas:
+    "header header"
+    "slogan image";
+}
+
+.landing-section-text p {
```

Comment:
You can put the p inside other landing section text. Same with h2.
.landing-section-text{
  p {}
  h2 {}
}
This could also be used with section-image

Reply:
Done 81b66da333131dec4711462a339d934eaad2c595